lib/http.js

```diff
@@ -56,10 +67,83 @@ async function main() {
       });
     })
   });
+
+  /**
+   * Wallet endpoints
+   */
+  app.post(`/wallets`, (req, res) => {
+    let accountId = req.body.accountId;
+
+    if(accountId == undefined) {
+      res.status(400).end();
+    }
+    else {
+      createEncryptedWallet(accountId).then(ret => {
+        res.status(201).end();
+      }).catch(e => {
+        console.log(e);
+        res.sendStatus(500);
+      });
+    }
+
+  });
+
+  app.get(`/wallets`, (req, res) => {
```

Comment:
when would we use this endpoint?

Reply:
mmm if for exp we want to show in the profile to the user info about his wallet?